deps/verifier/src/sample/mod.rs

```diff
@@ -80,7 +70,7 @@ fn parse_tee_evidence(quote: &SampleTeeEvidence) -> Result<TeeEvidenceParsedClai
     let claims_map = json!({
         "svn": quote.svn,
         "report_data": quote.report_data,
-        "init_data": quote.init_data,
+        "init_data": "",
```

Comment:
If we don't check the init-data, we shouldn't set this claim at all due to [this logic](https://github.com/confidential-containers/trustee/blob/main/attestation-service/src/token/ear_broker.rs#L429). The EAR broker will add the raw init-data to the token as long as the init-data claim is set.

Reply:
Thanks, fixed this.